wca-regulations.md

```diff
@@ -295,7 +295,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 11d) If the WCA Regulations are not fully clear or if the incident is not covered by the WCA Regulations, then the WCA Delegate must make a decision based on fair sportsmanship (also see [Regulation 11e3](regulations:regulation:11e3)).
 - 11e) If an incident occurs during an attempt, the WCA Delegate may grant a competitor an extra attempt, replacing the attempt during which the incident occurred. The competitor must appeal verbally or in writing to the judge and WCA Delegate at the time of the incident, before finishing the original attempt, to be eligible for an extra attempt. An appeal does not guarantee the competitor an extra attempt.
     - 11e1) If a competitor is granted an extra attempt, the extra attempt must be scrambled using a different scramble sequence. This scramble sequence must be generated using the current official version of the official WCA scramble program (see [Regulation 4f](regulations:regulation:4f)).
-    - 11e2) If a competitor receives any extra attempts, the regularly numbered attempts should be completed first, and extra attempts should be done afterwards. The results from the extra attempts should replace the regularly numbered attempts that caused them, in increasing order.
+    - 11e2) If a competitor is granted an extra attempt, the extra attempt should be done right after the attempt that caused it, and should replace the original regularly numbered attempt.
```

Comment:
It's not, from the style guide : 
* Attempt: The time frame starting when the competitor acknowledges his readiness to begin (see Regulation A3b2) and ending when the score sheet is complete (see Regulation A7c).

* Result: An official time, either the time for an attempt including penalties, or a derived result (e.g. mean/average).